README.md

```diff
@@ -30,7 +30,17 @@ individual tasks.
 Run is a single binary, which you can download from from the [releases
 page][releases].
 
-Alternately, if you already use go, you can install Run with the go command
+This repository provides a script to download and install the latest version of Run:
+
+    $ curl -sSfL https://raw.githubusercontent.com/amonks/run/main/install.sh | bash
+
+
+>[!IMPORTANT] 
+>**Security Notice**: Executing scripts from the internet can pose a significant security risk to your system. Before running the installation command, please take a moment to review the script to ensure its safety. By inspecting the script first, you can make an informed decision about whether to proceed with the installation.
```

Comment:
I acknowledge that a lot of people worry about this, but I don't think the worry is legitimate: the binary you download from the releases page could do just as much risky stuff, and nobody expects a warning there.

I think we should appease users who believe in this unreasonable double-standard by making the script as short and auditable as possible, but I don't think we should endorse their preferences with a highly visible warning.

Reply:
There seem to be a lot of opinions on the internet about curl|bash and warning users about it.

The opinion and reasoning you lay out here is the most sensible I've seen, and I agree with it completely :)